edk2toolext/versioninfo/versioninfo_helper.py

```diff
@@ -0,0 +1,556 @@
+## @file versioninfo_helper.py
+# This module contains helper functions for creating VERSIONINFO resources
+# from json files, along with the functions to output version information from
+# PE/PE+ files.
+#
+##
+# Copyright (C) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import pefile
+import sys
+import json
+import copy
+
+# String values for PE/PE+ header versioning metadata
+FILE_OS_STRINGS = {
```

Comment:
there is a lot of global data here.  Does it make sense to wrap this inside of a class?